spring-cloud-gcp-samples/spring-cloud-gcp-storage-resource-sample/src/test/java/com/example/GcsSampleApplicationIntegrationTests.java

```diff
@@ -58,6 +61,15 @@ class GcsSampleApplicationIntegrationTests {
   @Value("${gcs-resource-test-bucket}")
   private String bucketName;
 
+  @LocalServerPort private int port;
+
+  private String appUrl;
+
+  @BeforeEach
+  void initializeAppUrl() {
+    this.appUrl = "http://localhost:" + this.port;
+  }
+
   @BeforeEach
   @AfterEach
   void cleanupCloudStorage() {
```

Comment:
> Before and after each test, delete all blobs in the bucket.

If parallel invocations are running, this could delete blobs needed for actively running tests.